[3.2.4.rc1] Android custom build export failed on icons merge

<!-- Please search existing issues for potential duplicates before filing yours:
https://github.com/godotengine/godot/issues?q=is%3Aissue
-->

**Godot version:**
3.2.4.rc1.official

**OS/device including version:**
Windows 10

**Issue description:**
Export using custom build template failed:

Debug:

```
* What went wrong:
Execution failed for task ':processDebugResources'.
> Could not resolve all files for configuration ':debugRuntimeClasspath'.
   > Failed to transform godot-lib.debug.aar to match attributes {artifactType=android-compiled-dependencies-resources}.
      > Execution failed for AarResourcesCompilerTransform: C:\Users\poqxe\.gradle\caches\transforms-2\files-2.1\292c13cebff2ac459e69cf36eef145a8\jetified-godot-lib.debug.
         > AAPT2 aapt2-4.1.0-6503028-windows Daemon #0: Unexpected error during compile 'C:\Users\poqxe\.gradle\caches\transforms-2\files-2.1\292c13cebff2ac459e69cf36eef145a8\jetified-godot-lib.debug\res\mipmap\icon.png', attempting to stop daemon.
```

Release:

 ```
> Task :stripReleaseDebugSymbols
 Unable to strip the following libraries, packaging them as they are: libc++_shared.so, libgodot_android.so.
 
 > Task :mergeReleaseResources
 > Task :mergeReleaseResources FAILED
 
 FAILURE: Build failed with an exception.
 
 * What went wrong:
 Execution failed for task ':mergeReleaseResources'.
 > Multiple task action failures occurred:
    > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
       > AAPT2 aapt2-4.1.0-6503028-windows Daemon #0: Unexpected error during compile 'D:\Projects\Test324\android\build\res\drawable\splash_bg_color.png', attempting to stop daemon.
         This should not happen under normal circumstances, please file an issue if it does.
    > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
       > AAPT2 aapt2-4.1.0-6503028-windows Daemon #3: Unexpected error during compile 'D:\Projects\Test324\android\build\res\mipmap-xxxhdpi-v4\icon.png', attempting to stop daemon.
         This should not happen under normal circumstances, please file an issue if it does.
    > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
       > AAPT2 aapt2-4.1.0-6503028-windows Daemon #1: Unexpected error during compile 'D:\Projects\Test324\android\build\res\mipmap-xhdpi-v4\icon_foreground.png', attempting to stop daemon.
         This should not happen under normal circumstances, please file an issue if it does.
    > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
       > AAPT2 aapt2-4.1.0-6503028-windows Daemon #2: Unexpected error during compile 'D:\Projects\Test324\android\build\res\mipmap-xxhdpi-v4\icon_foreground.png', attempting to stop daemon.
         This should not happen under normal circumstances, please file an issue if it does.
    > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
       > AAPT2 aapt2-4.1.0-6503028-windows Daemon #4: Unexpected error during compile 'D:\Projects\Test324\android\build\res\mipmap\icon_foreground.png', attempting to stop daemon.
         This should not happen under normal circumstances, please file an issue if it does.
    > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
       > AAPT2 aapt2-4.1.0-6503028-windows Daemon #5: Unexpected error during compile 'D:\Projects\Test324\android\build\res\mipmap-xxhdpi-v4\icon.png', attempting to stop daemon.
         This should not happen under normal circumstances, please file an issue if it does.
    > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
       > AAPT2 aapt2-4.1.0-6503028-windows Daemon #6: Unexpected error during compile 'D:\Projects\Test324\android\build\res\drawable\splash.png', attempting to stop daemon.
         This should not happen under normal circumstances, please file an issue if it does.
    > A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
       > AAPT2 aapt2-4.1.0-6503028-windows Daemon #7: Unexpected error during compile 'D:\Projects\Test324\android\build\res\mipmap-hdpi-v4\icon_foreground.png', attempting to stop daemon.
         This should not happen under normal circumstances, please file an issue if it does.

```
**Steps to reproduce:**
1. Create project.
2. Install Android Build Template
3. Add Android export and set "Custom Build"
4. Setup keystore.
5. Export project and get failed


**Minimal reproduction project:**
Default project

> Both log files look normal. Looking at the debug error log in the issue description, it looks like the Godot engine was unable to convert `<USER_PATH>` into the corresponding filepath on your machine.

Sorry, it's incorrect copy-paste. Path in log is corrent `C:\Users\poqxe\.gradle\.....`

```
> Task :preBuild UP-TO-DATE
> Task :preDebugBuild UP-TO-DATE
> Task :compileDebugAidl NO-SOURCE
> Task :compileDebugRenderscript NO-SOURCE
> Task :generateDebugBuildConfig UP-TO-DATE
> Task :javaPreCompileDebug UP-TO-DATE
> Task :checkDebugAarMetadata UP-TO-DATE
> Task :generateDebugResValues UP-TO-DATE
> Task :generateDebugResources UP-TO-DATE
> Task :mergeDebugResources UP-TO-DATE
> Task :createDebugCompatibleScreenManifests UP-TO-DATE
> Task :extractDeepLinksDebug UP-TO-DATE
> Task :processDebugMainManifest UP-TO-DATE
> Task :processDebugManifest UP-TO-DATE
> Task :processDebugManifestForPackage UP-TO-DATE
> Task :processDebugResources FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':processDebugResources'.
> Could not resolve all files for configuration ':debugRuntimeClasspath'.
   > Failed to transform godot-lib.debug.aar to match attributes {artifactType=android-compiled-dependencies-resources}.
      > Execution failed for AarResourcesCompilerTransform: C:\Users\poqxe\.gradle\caches\transforms-2\files-2.1\292c13cebff2ac459e69cf36eef145a8\jetified-godot-lib.debug.
         > AAPT2 aapt2-4.1.0-6503028-windows Daemon #0: Unexpected error during compile 'C:\Users\poqxe\.gradle\caches\transforms-2\files-2.1\292c13cebff2ac459e69cf36eef145a8\jetified-godot-lib.debug\res\mipmap\icon.png', attempting to stop daemon.
           This should not happen under normal circumstances, please file an issue if it does.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 5s
11 actionable tasks: 1 executed, 10 up-to-date

Exit Code: 1
```

--- a/platform/android/java/app/config.gradle
+++ b/platform/android/java/app/config.gradle
@@ -1,5 +1,5 @@
 ext.versions = [
-    androidGradlePlugin: '4.1.0',
+    androidGradlePlugin: '4.0.1',
     compileSdk         : 29,
     minSdk             : 18,
     targetSdk          : 29,

